index.Rmd

```diff
@@ -373,27 +397,18 @@ More flexibility in your function can be achieved through some easy control flow
 Control flow refers to the use of conditions in your code that redirect the flow to different directions depending on variables values or class. Make use of that in your code, as this will make your functions more flexible and generic.
 
 ### Object classes and Control flow
-You have seen in a [previous tutorial](../RPythonBasics/index.html/#data-types) already that every variable in your R working environment belongs to a class. You can take advantage of that, using control flow, to make your functions more flexible.
-
-A quick reminder on classes:
+You have seen in a [previous tutorial](../RPythonBasics/index.html/#data-types) already that every variable in your *R* working environment belongs to a class. You can take advantage of that, using control flow, to make your functions more flexible. First, let's introduce a new class.
```

Comment:
Apart from this short part about the classes, I did not find much of tutorial 4 overlapping with tutorial 3 (RPythonBasics). I therefore just removed the very basic classes and kept the new SpatRaster class.